src/Symfony/Component/HttpKernel/EventListener/ResponseListener.php

```diff
@@ -46,6 +50,11 @@ public function onKernelResponse(ResponseEvent $event)
             $response->setCharset($this->charset);
         }
 
+        if ($this->addContentLanguageHeader && 1 < \count($this->enabledLocales) && !$response->isInformational() && !$response->isEmpty() && !$response->headers->has('Content-Language')) {
```

Comment:
What if we add a `Content-Language` header but the locale was not selected based on the `Accept-Language` header in LocaleListener ? The current condition looks weird to me.

Reply:
I think we don't care: `Accept-Language` and `Content-Language` are distinct headers, and `Content-Language` should always reflect the actual request locale when enabled, hence the two separate options. Makes sense?

Reply:
I removed the `count` part of the condition though, this listener doesn't care about enabled locales anymore